manifest.json

```diff
@@ -0,0 +1,12 @@
+{
+  "name": "Aragon Network",
+  "description": "Aragon Network Dashboard",
+  "icons": [
+    {
+      "src": "images/icon.png",
```

Comment:
(commenting here because we can’t on images)

We should update some files to use the new logo:

- `images/icon.png`
- `src/assets/favicon.png`
- `src/assets/favicon.svg`
- `src/assets/logo.png`